app/models/notification.rb

```diff
@@ -99,6 +108,14 @@ def conversation
     primary_actor
   end
 
+  def unread_count
```

Comment:
let's move these methods to user.rb since it doesn't make sense as a part of a notification object

```
def notifications_meta(account)
   { 
        unread_count: notifications.where(account_id: account.id, read_at: nil).count
   }
end
```